src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -910,12 +910,41 @@ if (typeof Sfjs === 'undefined' || typeof Sfjs.loadToolbar === 'undefined') {
                 document.querySelector('#log-filter-priority .filter-active-num').innerText = (priorities.length === selectedPriorities.length) ? 'All' : selectedPriorities.length;
                 document.querySelector('#log-filter-channel .filter-active-num').innerText = (channels.length === selectedChannels.length) ? 'All' : selectedChannels.length;
             },
+
+            convertDateTimesToUserTimezone: function() {
+                const userTimezoneName = Intl.DateTimeFormat().resolvedOptions().timeZone;
+
+                document.querySelectorAll('time[data-convert-to-user-timezone]').forEach((timeElement) => {
+                    const iso8601Datetime = timeElement.getAttribute('datetime');
+                    const dateInUserTimezone = new Date(iso8601Datetime);
+
+                    if (timeElement.hasAttribute('data-render-as-datetime')) {
+                        timeElement.innerHTML = dateInUserTimezone.toLocaleString(undefined, {
+                            year: 'numeric', month: 'long', day: 'numeric',
+                            hour: 'numeric', minute: 'numeric', second: 'numeric'
+                        });
+                    } else if (timeElement.hasAttribute('data-render-as-date')) {
+                        timeElement.innerHTML = dateInUserTimezone.toLocaleString(undefined, {
+                            year: 'numeric', month: 'long', day: 'numeric'
+                        });
+                    } else if (timeElement.hasAttribute('data-render-as-time')) {
+                        timeElement.innerHTML = dateInUserTimezone.toLocaleString(undefined, {
+                            hour: 'numeric', minute: 'numeric', second: 'numeric'
+                        });
+                    }
+
+                    if (undefined !== userTimezoneName) {
+                        timeElement.setAttribute('title', `Date/times shown in timezone: ${userTimezoneName}`);
```

Comment:
this should be done only if we don't already have a title attribute IMO.

Reply:
Yes. Fixed.